runtime/collections/kelemle.cpp

```diff
@@ -125,6 +125,25 @@ bool hook_KEQUAL_eq(block *arg1, block *arg2) {
           }
           break;
         }
+        case MINT_LAYOUT + 32: {
```

Comment:
Could you remind me where is the code that constructs the MINT layouts following this convention about how to find the bit width?

Reply:
I don't remember exactly where that code lives, but basically, when we have to encode a sort category as an integer, the bit width gets added to the enum value. I tested this and it works.